internal/debug/debug.go

```diff
@@ -22,4 +22,6 @@ const (
 	// TAPPortPacketTrace turns on packet tracing for lemming's TAP interface ports
 	// that communicate between each external port and lemming's tasks.
 	TAPPortPacketTrace = true
+	// SysRIB controls whether debug messages for the main system RIB are printed.
+	SysRIB = false
```

Comment:
maybe add TODO for use viper for this you don't have change src code

Reply:
Do you mean so that the flags don't all have to be added to this file, and by changing topology.txtpb we get access to different systems' debug messages?

Reply:
yes